src/server/pfs/server/driver.go

```diff
@@ -953,16 +943,18 @@ func (d *driver) repoSize(ctx context.Context, txnCtx *txncontext.TransactionCon
 			if err != nil {
 				return 0, errors.Wrap(err, "repo size")
 			}
-			commit := branchInfo.Head
+			commit, err := d.resolveCommit(ctx, txnCtx.SqlTx, branchInfo.Head)
```

Comment:
[nit] the resolveCommit can be renamed, as the resolveCommitInfo is also renamed.